tests/TestCase.php

```diff
@@ -11,17 +11,20 @@
 
 abstract class TestCase extends BaseTestCase
 {
+    private const DRIVER_CONFIG_FACTORY_KEY = 'driver_config_factory';
+    private const MINK_SESSION_KEY = 'sess';
+
     /**
      * Mink session manager.
      *
-     * @var Mink
+     * @var Mink|null
      */
-    private static $mink;
+    private static $mink = null;
```

Comment:
`= null` is useless when the property is not typed, as it is already the case implicitly